config/secrets.yml.sample

```diff
@@ -0,0 +1,39 @@
+default: &default
+  database:
+    name: datarepo
+    username: vagrant
+    password: changeme
+  ezid:
+    default_shoulder: doi:10.5072/FK2
+    user: apitest
+    password: apitest
+  fedora:
+    url: http://127.0.0.1:8080/fedora/rest
+    user: fedoraAdmin
+    password: fedoraAdmin
+  orcid:
+    app_id: 0000-0000-0000-0000
+    app_secret: XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX
+  redis:
+    host: localhost
+    port: 6379
+  cas_endpoint_url: https://cas-dev.middleware.vt.edu
+development:
+  <<: *default
+  fedora:
+    base_path: /dev
+  blacklight_url: http://127.0.0.1:8983/solr/development
+  solr_url: http://localhost:8983/solr/development
+test:
+  <<: *default
+  fedora:
+    base_path: /test
```

Comment:
For RAILS_ENV="test" we should have "url: http://127.0.0.1:8983/fedora/rest" because we are using hydra-jetty in the test environment.